components/modal/Confirm.js

```diff
@@ -35,8 +35,8 @@ Confirm.propTypes = {
     onConfirm: PropTypes.func,
     title: PropTypes.string.isRequired,
     children: PropTypes.node.isRequired,
-    cancel: PropTypes.string,
-    confirm: PropTypes.string,
+    cancel: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
```

Comment:
If you use `PropTypes.node`, it will support also `String` as input.
```suggestion
    cancel: PropTypes.node,
```

Reply:
Yeah, that's true. I thought it was more clear to be that explicit (we do that in some other places). But there's no point in being redundant. Just having `node` now